webapp/channels/src/components/user_group_popover/user_group_popover.tsx

```diff
@@ -209,9 +207,9 @@ const UserGroupPopover = (props: Props) => {
                             onClick={handleClose}
                             ref={closeRef}
                         >
-                            <LocalizedIcon
+                            <i
                                 className='icon icon-close'
-                                ariaLabel={{id: t('user_group_popover.close'), defaultMessage: 'Close'}}
+                                aria-label={formatMessage({id: 'user_group_popover.close', defaultMessage: 'Close'})}
```

Comment:
@sapnasivakumar could you remove aria label from here because the button already contains the aria label.

Reply:
@M-ZubairAhmed Done